lib/inc/sairedis.h

```diff
@@ -70,4 +70,137 @@ typedef enum _sai_redis_switch_attr_t
 
 } sai_redis_switch_attr_t;
 
+/*
+ * Those bulk APIs are provided as static functions since current SAI 0.9.4
+ * doesn't support bulk API.  Also note that SDK don't support bulk route API
+ * yet, so those APIs are introduced here to reduce number of calls to redis,
+ * in syncd bulk API will be splitted to separate call's until proper SDK
+ * support will be added.
+ *
```

Comment:
Nice to have this feature at redis-db level. But what about SDK level? Are you aware of any upcoming SAI proposal to address bulk requests?